scripts/generate_upgrade_syncset.py

```diff
@@ -0,0 +1,301 @@
+#!/bin/env python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import ruamel.yaml as yaml
+from ruamel.yaml.scalarstring import DoubleQuotedScalarString as dq
```

Comment:
Not used

Reply:
@2uasimojo It is. See lines 119, 124, 125, 161, 202. I'm curious why it's used and what it does.